gym/wrappers/pixel_observation.py

```diff
@@ -0,0 +1,111 @@
+"""An observation wrapper that augments observations by pixel values."""
+
+import collections
+import copy
+
+import numpy as np
+
+from gym import spaces
+from gym import ObservationWrapper
+
+STATE_KEY = 'state'
+
+
+class PixelObservationWrapper(ObservationWrapper):
+    """Augment observations by pixel values."""
+    def __init__(self,
```

Comment:
I think a mode that does not use dictionary observations and returns only pixels would also be useful

Reply:
I think this might not be necessary since you could use the `FlattenDictWrapper`. What do you think?